src/OSPSuite.UI/Views/Importer/ImporterView.cs

```diff
@@ -32,18 +27,32 @@ public override void InitializeResources()
          previewLayoutControlItem.Name = Captions.Importer.PreviewLayout;
          columnMappingLayoutControlItem.Name = Captions.Importer.MappingSettings;
 
-         saveMappingBtn.Click += (s, a) => OnEvent(() =>
+         saveMappingBtn.Click += (o, e) => OnEvent(() =>
```

Comment:
all of those can be on one line
saveMappingBtn.Click  += (o,e) => OnEvent(() => _presenter.SaveConfiguration());

or even

saveMappingBtn.Click  += (o,e) => OnEvent(_presenter.SaveConfiguration);

Reply:
done